src/daemon/Display.cpp

```diff
@@ -499,6 +514,11 @@ namespace SDDM {
     void Display::slotAuthError(const QString &message, Auth::Error error) {
         qWarning() << "Authentication error:" << error << message;
 
+        if (m_auth->autologin()) {
+            handleAutologinFailure();
+            return;
```

Comment:
What happens if you add `m_socketServer->informationMessage(m_socket, message);` after the `handleAutologinFailure()`? My hope is that the greeter will react to it like a manual login failed, giving some indication to the user

Reply:
That's crashed. Because `m_socket == nullptr`. If indication of failed login in autologin case is important, we can create bool variable `m_autologinFailed` in `Display` and connect to `SocketServer::connected` signal (that signal also have to pass `socket` pointer, like `login` signal do). And it will be like:
```
void Display::slotSocketServerConnected(QLocalSocket *socket) {
        m_socket = socket;

        if (m_autologinFailed && m_socket)
            emit loginFailed(m_socket);
}
```
I locally tried that and it's work, but I'm not sure that is 100% proper solution. I think maybe passing `socket` in `connected` signal will be even more relevant than passing this through `login` signal, but I don't know all the subtleties of this.